src/services/textChanges.ts

```diff
@@ -632,7 +632,7 @@ namespace ts.textChanges {
         }
 
         private computeSpan(change: Change, _sourceFile: SourceFile): TextSpan {
-            return createTextSpanFromBounds(change.range.pos, change.range.end);
+            return createTextSpanFromRange(change.range);

```

Comment:
Is this name the intended one? On other files the method is creatTextSpanFromStartEnd.

Reply:
This and `createTextSpanFromStartEnd` are basically the same functions, only some data structures use `{ start, end }` and others use `{ pos, end }` for some reason.

Reply:
If both methods (FromStartEnd and FromRange) achieves the same goal, is it preferable to instead refactor to use the FromRange and not introduce a new method?

Reply:
:+1: Changed this so that they call it `pos` consistently